src/game/server/neo/neo_player.cpp

```diff
@@ -3231,3 +3261,233 @@ void CNEO_Player::ModifyFireBulletsDamage(CTakeDamageInfo* dmgInfo)
 {
 	dmgInfo->SetDamage(dmgInfo->GetDamage() * sv_neo_wep_dmg_modifier.GetFloat());
 }
+
+// Start spectator takeover of player related code:
+ConVar sv_neo_spec_replace_player_bot_enable("sv_neo_spec_replace_player_bot_enable", "1", FCVAR_CHEAT, "Allow spectators to take over bots.");
+ConVar sv_neo_spec_replace_player_afk_enable("sv_neo_spec_replace_player_afk_enable", "1", FCVAR_CHEAT, "Allow spectators to take over AFK players.");
+ConVar sv_neo_spec_replace_player_afk_time_sec("sv_neo_spec_replace_player_afk_time_sec", "300", FCVAR_CHEAT, "Seconds of inactivity before a player is considered AFK.");
```

Comment:
These probably don't need to be marked as cheats, so servers can disable/edit the feature if they want to, without requiring turning on `sv_cheats`